concourse-ci/pipelines/build-deno-pipeline.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       #   inputs: [app]
       #   params: {state: pending}
       - load_var: app_version
-        file: app/backend/version.json
+        file: app/backend/VERSION
```

Comment:
KISS in action right here, sha + timestamp info can get fetched from git log if needed, let's stick with a VERSION file containing only one thing and adapt if we need to later